Core/src/GrpcTrait.php

```diff
@@ -96,12 +96,15 @@ public function send(callable $request, array $args, $whitelisted = false)
      * @param callable|null $authHttpHandler
      * @return array
      */
-    private function getGaxConfig($version, callable $authHttpHandler = null)
+    private function getGaxConfig(
+        $version,
+        callable $authHttpHandler = null,
+        string $transport = 'grpc')
     {
         $config = [
             'libName' => 'gccl',
             'libVersion' => $version,
-            'transport' => 'grpc'
+            'transport' => $transport
```

Comment:
I am following the reason why this is done - it's a lot more elegant than rewriting everything. However, I think it's a bit odd to pass a different transport name than `grpc` into the `GrpcTrait`.

I think we can do this a bit cleaner - see my suggestion below